bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/BusAlarmZone.xml

```diff
@@ -17,6 +17,7 @@
 			<!-- read only -->
 			<channel id="state" typeId="alarmZoneState"/>
 			<channel id="alarm" typeId="zoneAlarm"/>
+			<channel id="timestamp" typeId="zoneAlarmTimeStamp"/>
```

Comment:
```suggestion
			<channel id="timestamp" typeId="zoneAlarmTimestamp"/>
```
Timestamp and not TimeStamp (it's one word)

Reply:
done